README.md

```diff
@@ -89,6 +89,8 @@ The below figurs illusrates the
 
 Once you have installed, start MLModelCI service on a leader server by:
 ```shell script
+# setup environment
+source scripts/setup_env.sh
```

Comment:
If we are replacing all DB code with pymongo, why still needs this source to setup the system?

Reply:
Will be switch to hardcode by default